### ISSUE TITLE
Create DMARC TXT record for semantic-mediawiki.org

### ISSUE THREAD

**Issue by kghbln** [SemanticMediaWiki/semantic-mediawiki.org#32]
Reported [here](https://www.semantic-mediawiki.org/wiki/Thread:User_talk:Wladek92/Email). Some e-mail providers reject e-mails coming from domains without a DMARC policy. So we probably have to cater for this bogus.

Resources:
* https://en.wikipedia.org/wiki/DMARC
* https://dmarc.org/resources/deployment-tools/
* https://dmarcguide.globalcyberalliance.org/#/dmarc?lang=en
* http://www.trusteddomain.org/opendmarc.html

**Comment by kghbln** [comment]
@JeroenDeDauw You will love this.

**Comment by kghbln** [comment]
Note: Setting [`$wgUserEmailUseReplyTo = true;`](https://github.com/wikimedia/mediawiki/commit/dfb50ab9f3e119f4bd3425863068406c111a988d) seems not to have had the desired effect, though ... see below. The WMF however does not use this either. Perhaps a tough titties situation for users of Gmail, Yahoo, AOL and consorts.

I asked for a fresh test to verify if the setting actually changed something.

**Comment by JeroenDeDauw** [comment]
Should I do something on our hosting provider?

**Comment by kghbln** [comment]
> Should I do something on our hosting provider?

I am afraid that we need do define this policy for the s-mw.o domain. People trying to register using Gmail, Yahoo etc. just do not get their e-mail-confirmation requests either because they are bounced or filtered away.

Unconfirmed valid requests are still happening.

**Comment by mkroetzsch** [comment]
So what should I do to fix this?

**Comment by mkroetzsch** [comment]
Can I just add a DNS entry like this: https://help.united-domains.de/faq-article/was-ist-ein-dmarc-eintrag

It seems that the main challenge is to decide which email to provide for error reporting (ruf), as this could receive huge amounts of emails. Any suggestions?

**Comment by kghbln** [comment]
> Can I just add a DNS entry like this: https://help.united-domains.de/faq-article/was-ist-ein-dmarc-eintrag

Yes, I believe so. I know that Hetzner generates such entries by default which only need to be activated at some point.

> It seems that the main challenge is to decide which email to provide for error reporting (ruf), as this could receive huge amounts of emails. Any suggestions?

Perhaps <postmaster@...> which could be checked once in a while.

This task is not a front burner since we survived without during the past years. However I expect increased user friendliness for our mails no longer ending up in Googles spam folders (most registrations come from Gmail accounts. :( )

**Comment by kghbln** [comment]
Probably useful: https://mxtoolbox.com/DMARCRecordGenerator.aspx
